model/ent/schema/deployment.go

```diff
@@ -26,6 +26,8 @@ func (Deployment) Fields() []ent.Field {
 			Default("commit"),
 		field.String("env"),
 		field.String("ref"),
+		field.JSON("dynamic_payload", map[string]interface{}{}).
+			Optional(),
```

Comment:
We should add the `dynamic_payload` field to send the same payload after review process.